### PR TITLE
Implement double differential distributions in ATLAS_TTBAR_13TEV_LJ_DIF

### PR DESCRIPTION
**Pull request by t7phy**
This PR implements `ATLAS_TTBAR_13TEV_LJ_DIF_MTTBAR-PTT`, `ATLAS_TTBAR_13TEV_LJ_DIF_MTTBAR-PTT-NORM`, `ATLAS_TTBAR_13TEV_LJ_DIF_PTT-YT` and `ATLAS_TTBAR_13TEV_LJ_DIF_PTT-YT-NORM`. The covariance matrices for `PTT-YT` and `PTT-YT_NORM` are bugged, however, they are left in the data implementation for completeness (and in case they are corrected by the experimentalists, re-downloading the hepdata files and rerunning the filter should be enough). The data-theory comparison is available here: https://vp.nnpdf.science/Eu-MC_FSR_-iGv7_gij3XA==

The grids for these datasets will be added to nnpdf/theories_slim#43

### CODE REVIEW

File: nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml [hunk 1]
```diff
@@ -244,3 +244,121 @@ implemented_observables:
...
+    observable_name: PTT-YT
+    process_type: HQP_PTQ
+    ndata: 13
+    tables: [649, 650, 651]
+    kinematics:
+      variables:
+        pT_t: {description: "transverse momentum of t", label: "$pT_t$", units: "$GeV$"}
+        y_t: {description: "absolute rapidity of t", label: '$|y_{t}|$', units: ""}
+        m_t2: {description: "top mass squared", label: '$m_{t}^2$', units: '$GeV^2$'}
+        sqrts: {description: "center of mass energy", label: '$\sqrt{s}$', units: 'GeV'}
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml:258]: Hi @t7phy I think this is good to go. The only comment is that I think it is better to remove the `sqrts`. It just makes thing heavier and the information is in the title of the dataset anyway (and it is accessible by validphys)

File: nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml [hunk 2]
```diff
@@ -12,8 +12,8 @@ hepdata:
   url: "https://www.hepdata.net/record/ins1750330"
   version: 1
 
-version: 2
-version_comment: "Add uncertainty variant without luminosity uncertainty"
+version: 3
+version_comment: "Implement double differential distributions"
```
Comment (scarlehoff) [inline nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml:16]: Add the comment about the bugs here, this is the kind of thing that makes sense to keep as a comment!

**Review by scarlehoff — APPROVED** [review]
lgtm, just the two minor comments

File: nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml (hunk 2 above)
Comment (t7phy) [reply nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml]: https://github.com/NNPDF/nnpdf/pull/2218/commits/f48776ae1b2b674c55a23821dab28422d8efd6c4

File: nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml (hunk 1 above)
Comment (t7phy) [reply nnpdf_data/nnpdf_data/commondata/ATLAS_TTBAR_13TEV_LJ_DIF/metadata.yaml]: https://github.com/NNPDF/nnpdf/pull/2218/commits/977b4a17c5bae94a2c1cb8f9202fcc22ada072dd

**Pull request merged**